apps/course-api/package.json

```diff
@@ -11,7 +11,8 @@
     "integration": "docker compose -f docker-compose.integration.yaml down -v && docker compose -f docker-compose.integration.yaml up --build --force-recreate --abort-on-container-exit --exit-code-from app",
     "integration:jest": "jest test/integration --detectOpenHandles",
     "docker:test_build": "docker compose -f docker-compose.test-build.yaml down -v && docker compose -f docker-compose.test-build.yaml up -d --build --force-recreate",
-    "prisma:generate_types": "pnpm prisma generate --schema=../../packages/prisma/schema.prisma"
+    "bootstrap:unix": "pnpm prisma generate --schema=../../packages/prisma/schema.prisma",
+    "bootstrap:windows": "pnpm prisma generate --schema=../../packages/prisma/schema.prisma"
```

Comment:
??

Reply:
it's an attempt to generate prisma for course api automatically when running either `pnpm run bootstrap:unix` or `pnpm run bootstrap:windows`. (The commands must be the same for turbo to detect, unless there's a shortcut for this XD)